### PR TITLE
Add support for ARIA attributes (props)

### PR DESCRIPTION
**Pull request by gasi**
This will allow us to improve accessibility of certain features, e.g. breadcrumbs from Bootstrap: https://getbootstrap.com/docs/4.6/components/breadcrumb/

### CODE REVIEW

File: codegen/html.js [hunk 1]
```diff
@@ -36,6 +36,9 @@ const printRow = (e, elProps) =>
   elProps.length > 0
     ? `
   ( _data :: Object String
+  -- ARIA
+  , ${ariaProps.map(p => `${p} :: String`).join("\n  , ")}
```
Comment (fsoikin) [inline codegen/html.js]: These are invalid record labels. You have to double-quote them:

```suggestion
  , ${ariaProps.map(p => `"${p}" :: String`).join("\n  , ")}
```

Comment (gasi) [reply codegen/html.js]: Right, I haven’t yet had a chance to test this, but will before merging 👍

**Review by fsoikin — APPROVED** [review]

**Review by vineshay — APPROVED** [review]
I've tested this locally and the attributes are able to be used and work as expected

**Review by fsoikin — APPROVED** [review]
👍

**Pull request merged**